tests/snapshot/test_lido_snapshot.py

```diff
@@ -32,12 +32,26 @@ class Frame(TypedDict):
 
 UINT256_MAX = 2**256 - 1
 _1ETH = Wei(10**18)
+ZERO_BYTES32 = b'\x00' * 32
+
 
 EXPECTED_SNAPSHOT_DIFFS: dict[str, Any] = {
     "canPerform()": (True, False),
-    "getRecoveryVault": (AGENT, ZERO_ADDRESS)
+    "getRecoveryVault": (AGENT, ZERO_ADDRESS),
```

Comment:
These two snapshot diffs seems outdated and left from the previous vote. In the current vote there should be no diffs for this methods